azkaban-sql/src/sql/create.execution_flows_status.sql

```diff
@@ -0,0 +1,25 @@
+-- -----------------------------------------------------------------
+-- Create lookup table for 'status' values in execution_flows table
+-- -----------------------------------------------------------------
+CREATE TABLE if NOT EXISTS execution_flows_status (
+	name VARCHAR(64),
+	status INT,
+	PRIMARY KEY (name)
+);
+-- -----------------------------------------------------------------
+-- Populate enum values into lookup table
```

Comment:
Consider adding a comment to the code where the enum is defined that this sql script needs to be updated when the enum is changed. And there is no automated test due to the low return on investment.